### ISSUE TITLE
Proxy running but clients can't connect

### ISSUE THREAD

**Issue by madl-ash** [TelegramMessenger/MTProxy#132]
like issue #79,
i run server using : 
`./mtproto-proxy -u nobody -p 8888 -H 443 -S client_secret -P promotion_tag --aes-pwd proxy-secret proxy-multi.conf -M 1`

but i can't connect to proxy,
server output:
```
[9186][2018-06-12 15:58:50.285368 local] Invoking engine mtproxy-0.01 compiled at Jun 12 2018 xx:xx:xx by gcc 7.3.0 64-bit after commit 20824f5fa8af3939e667fba5c1785f02059c1c43
[9186][2018-06-12 15:58:50.285658 local] config_filename = 'proxy-multi.conf'
[9186][2018-06-12 15:58:50.285914 local] creating 1 workers
[9186][2018-06-12 15:58:50.286788 local] Started as [xx]
[9186][2018-06-12 15:58:50.287223 local] configuration file proxy-multi.conf re-read successfully (797 bytes parsed), new configuration active
[9186][2018-06-12 15:58:50.287448 local] main loop
[9187][2018-06-12 15:58:50.288817 local] Started as [xx]
[9187][2018-06-12 15:58:50.289367 local] configuration file proxy-multi.conf re-read successfully (797 bytes parsed), new configuration active
[9187][2018-06-12 15:58:50.289490 local] main loop
```
and it stucks at this point.what should i do?the port is open

**Comment by loskiq** [comment]
Probably your proxy server is behind the NAT of your ISP.

**Comment by madl-ash** [comment]
@loskiq i don't think thats the case.if i were behind the NAT of the ISP,other ports and apps must have been the same way.i have lots of other things running perfectly but i guess this is thr problem of mtproto-proxy.on another server i used the docker image and it was working

**Comment by VahidPY** [comment]
Maybe the problem is your VPS provider as discussed at https://github.com/TelegramMessenger/MTProxy/issues/87#issuecomment-394874760. I don't think it is blocked from inside the country (if you cannot even connect through VPN from outside).

**Comment by madl-ash** [comment]
@VahidPY i use hetzner and until couple of days ago it was okay so i dont think thats the reason

**Comment by VahidPY** [comment]
@madl-ash Still you need to test more. Can you send request to telegram servers through your server by `curl` or `ping`?
@p1ratrulezzz Have you received reports about Hetzner?
Everyone who has problem, thinks mtproto is blocked automatically, but how does my proxy server work perfectly after a week with 7 users? I don't use any of the popular VPS providers.
See https://www.lowendtalk.com/discussion/142577 which says 'Russia bans Google Cloud, Amazon, Azure, Digital Ocean, Online.net, Hetzner, OVH, others', so perhaps they are blocking telegram connections to avoid making problem for other users.

**Comment by madl-ash** [comment]
@VahidPY i can access telegram servers.tested another proxy https://github.com/alexbers/mtprotoproxy and worked perfectly.

**Comment by p1ratrulezzz** [comment]
> @p1ratrulezzz Have you received reports about Hetzner?

No,in Russia hetzner servers are working. But yes, it happened once, they have blocked few millions of ipv4 addresses and broke all the internet (banks, viber, whatsapp and other services were broken). Playstation Network still has issues because of this. While Iran's approach for blocking is to use DPI, Russia just blocked all the internet :) For now OVH and Hetzner and even Amazon were unblocked.

**Comment by force-net** [comment]
I've found an issue, that if VPS has non-synced time (3 minute difference in my case), it causes such error in logs.

**Comment by ParaNULL** [comment]
had the same problem,the problem has beem solved after installing and configuring ntp on my deb server...

**Comment by lequne** [comment]
> 有同样的问题，在我的deb服务器上安装和配置ntp后问题已经解决了...

teach me

**Comment by mexsser** [comment]
For Debian based VPS, you can refer to [https://wiki.debian.org/DateTime](https://wiki.debian.org/DateTime) on how to setup ntp service on your VPS. I have solved this problem by installing and configuring ntp service.

**Comment by alehdaghi** [comment]
I have this issue yet! Could anybody solve this?

**Comment by LarinLive** [comment]
Same issue. Any idea after six years?
